src/AWS.Distro.OpenTelemetry.AutoInstrumentation/AwsSpanProcessingUtil.cs

```diff
@@ -52,6 +52,10 @@ internal sealed class AwsSpanProcessingUtil
     // If we add a longer keyword to the sql dialect keyword list, need to update the constant below.
     internal static readonly int MaxKeywordLength = 27;
 
+    // TODO: remove once supported by Semantic Conventions
+    internal static readonly string AttributeGenAiModelId = "gen_ai.request.model";
+    internal static readonly string AttributeGenAiSystem = "gen_ai.system";
```

Comment:
These 2 attributes have been defined in `AWSSemanticConventions.cs` as public vars, why do you need to define it here again?

Reply:
Good catch, I added them to AwsSpanProcessingUtil for feature parity with Python (addressing [this comment](https://github.com/aws-observability/aws-otel-dotnet-instrumentation/pull/73#discussion_r1718998133)) but didn't remove from AWSSemanticConventions.cs, its fixed now.